pkg/names/names.go

```diff
@@ -115,6 +117,34 @@ var (
 	}
 )
 
+var goKeyWords = []string{
```

Comment:
femto-nit: should be `goKeywords` since "keyword" is a single word. :)

Reply:
Haaa TIL 👍